src/hello.cpp

```diff
@@ -0,0 +1,10 @@
+#include "hello.hpp"
+
+// Standalone function (because it's not a class)
+NAN_METHOD(hello) {
+
+    // info comes from the NAN_METHOD macro, which returns differently
+    // according to the version of node
+    info.GetReturnValue().Set(Nan::New<v8::String>("world").ToLocalChecked());
+
+}
```

Comment:
We could absolutely add another function here in the same cpp file. The concept of `standalone function` only relates to "let's have a function that is not a method on a class, but rather on its own". In this case the `standalone function` would be in the global scope. If namespaces were used it would be in the namespace (http://en.cppreference.com/w/cpp/language/namespace).

In fact I think it would be a good idea to drop this in a namespace from the start. That would look like:

 - Adding `namespace <some name> { ... }`
 - Putting the function inside the namespace
 - Changing the `Nan::SetMethod(target, "hello", hello);` in `src/module.cpp` to `Nan::SetMethod(target, "hello", somename::hello);`